src/tui.rs

```diff
@@ -1,22 +1,23 @@
 use std::{
```

Comment:
please revert all changes in `tui.rs` - all files that are part of `ratatui` library should be leaved alone please